templates/login/index.html.twig

```diff
@@ -3,19 +3,36 @@
 {% block title %}Connexion{% endblock %}
 
 {% block body %}
-    {% if error %}
-        <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
-    {% endif %}
 
-    <form action="{{ path('login') }}" method="post">
-        <label for="username">Email:</label>
-        <input type="text" id="username" name="_username" value="{{ last_username }}"/>
+   {{ include('components/_header.html.twig') }}
+    <div class="container">
+        <div class="img-user">
+            <img src="{{ asset('build/images/user.png') }}">
+        </div>
+        {% if error %}
+            <div class="error">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+        {% endif %}
+        <form action="{{ path('login') }}" method="post">
+            <div class="form-class">
+                <input type="text" id="username" class="email" name="_username" placeholder="Adresse mail" value="{{ last_username }}"/>
+            </div>
+            <div class="form-class">
+                <input type="password" class="password" id="password" name="_password"/>
+            </div>
+            <input type="hidden" name="_target_path" value="/"/>
 
-        <label for="password">Password:</label>
-        <input type="password" id="password" name="_password"/>
+            <div class="submit">
+                <button type="submit" class="btn login">Je me connecte</button>
+            </div>
+            <div class="forgot">
+                <a href="#"><p>Mot de passe oublié ?</p></a>
+            </div>
+            <div class="not-account">
+                <a href="#"><p>Vous n'avez pas de compte</p></a>
+            </div>
+        </form>
```

Comment:
Your login form is not CSRF protected, you probably should.
C.f. : https://symfony.com/doc/current/security.html#csrf-protection-in-login-forms